.ddev/config.yaml

```diff
@@ -13,11 +13,10 @@ provider: default
 upload_dir: "media/files, media/images" # Specify the directories containing user-generated uploads
 use_dns_when_possible: true
 composer_version: "2"
-webimage_extra_packages: [php8.3-imap]
+webimage_extra_packages: [php8.2-imap]
```

Comment:
Seems we go back to PHP 8.2 for ddev

Reply:
Yep, this merge is bad. Possibly due to the revert merge in the history. I'll prepare another one where I'll merge PR after PR instead.